clusterloader2/pkg/measurement/common/probes/manifests/dns-prober-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+{{$ENABLE_DNS_PROBES := DefaultParam .ENABLE_DNS_PROBES false}}
```

Comment:
I'm wondering if it wouldn't be better to move it to test configs (similarly as this is done here:
https://github.com/kubernetes/perf-tests/pull/719/files)

But maybe it's not that big difference, because we will remove it in couple weeks anyway, right?

Reply:
Not easily, unfortunately. probes.go is implemented as monolith and doesn't have any knobs to turn on or off specific queries/probes. I've pointed that earlier, when I wanted to refactror it, so we'd have a single measurement per-probe but you were against it at that time.

Reply:
Saying so, I'd rather have a measurement per-prober. This way we can easily enable and disable existing probes. Most of the existing code could be reused, in a way that adding new measuremetn would be a few line of code, similarly to how [Prometheus-based measurements were implemented](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go)